build/docker/kubeone.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
I'd prefer having this in the README instead of the script or maybe having both. The scripts are also not platform-agnostic, i.e. those are going to work on Linux and macOS but not on Windows.

Reply:
Right you are. On it